src/platform/web/index.ts

```diff
@@ -27,7 +26,9 @@ Platform.Config = Config;
 Platform.Transports = Transports;
 Platform.WebStorage = WebStorage;
 
-BaseClient.Crypto = Crypto;
+for (const clientClass of [DefaultRest, DefaultRealtime]) {
+  clientClass.Crypto = Crypto;
```

Comment:
Would it not make more sense to just export `Crypto` separately from v2 onwards?

Reply:
How would the non-tree-shakable version of the library then access crypto functionality?

Reply:
I was thinking in `defaultrest.ts` and `defaultrealtime.ts` just import the crypto class directly?

Reply:
The crypto class varies by platform, though — how would those files know which one to import?

Reply:
ah yep you're right, okay let's leave it like this then 👍